lib/fluent/plugin/exception_detector.rb

```diff
@@ -16,7 +16,7 @@ module Fluent
   Struct.new('Rule', :from_state, :pattern, :to_state)
 
   # Configuration of the state machine that detects exceptions.
-  module ExceptionDetectorConfig
+  module ExceptionDetectorConfig # rubocop:disable Metrics/ModuleLength
```

Comment:
Seems that we've increased `Metrics/MethodLength`. This disable is no longer needed then?

Reply:
I've dropped this, it is now passing CI.